tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -198,29 +198,18 @@ def test_file_not_found_munge_frame_fallback_success(self, mock_integration, moc
         assert response.data["sourceUrl"] == "https://github.com/repo/path/to/munged/file.py"
         assert response.data["integrations"] == [serialized_integration(self.integration)]
 
-    @mock.patch("sentry.api.endpoints.project_stacktrace_link.munged_filename_and_frames")
```

Comment:
Dropping it since the test does not match any code mapping and we should not be attempting to munge.